tests/ztimer_underflow/main.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 SSV Software Systems GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    ztimer test application
+ *
+ * @author   Juergen Fitschen <jfi@ssv-embedded.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "ztimer.h"
+
+#ifndef TEST_ZTIMER_CLOCK
+#error Please define TEST_ZTIMER_CLOCK
+#endif
+
+#define xstr(s) str(s)
+#define str(s) #s
+
+int main(void)
+{
+    unsigned i = 0;
+
+    printf("Looping over ztimer clock %s\n", xstr(TEST_ZTIMER_CLOCK));
+    while (true) {
```

Comment:
IMO it would be nice if this is somehow bounded. "run for a couple of seconds" is quite arbitrary.

Maybe add ITERATIONS, after which it would exit and print a success message? Default that to maybe a second or so, depending on the clock used?

Reply:
of course the iterations are in the test script, justs saw that. Maybe having them here is preferable? better to have a thousand "loop ..." and then a success, than unlimited "loop...", when looking at this manually.

Reply:
I pushed a fixup. It stops after 1000 iterations. The test case just waits for "SUCCESS!".